File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/generic/CmdSetMoodInternal.java
package com.projectswg.holocore.resources.support.global.commands.callbacks.generic;

import com.projectswg.holocore.resources.support.global.commands.ICmdCallback;
import com.projectswg.holocore.resources.support.global.player.Player;
import com.projectswg.holocore.resources.support.objects.swg.SWGObject;
import org.jetbrains.annotations.NotNull;

public final class CmdSetMoodInternal implements ICmdCallback {
	
	@Override
	public void execute(@NotNull Player player, SWGObject target, @NotNull String args) {
		if (args.indexOf(' ') != -1) {
			return;
		}
		
		player.getCreatureObject().setMoodId(Byte.valueOf(args));
	}
	
}

File: src/main/java/com/projectswg/holocore/resources/support/global/commands/callbacks/generic/CmdSetMoodInternal.kt
package com.projectswg.holocore.resources.support.global.commands.callbacks.generic

import com.projectswg.holocore.resources.support.global.commands.ICmdCallback
import com.projectswg.holocore.resources.support.global.player.Player
import com.projectswg.holocore.resources.support.objects.swg.SWGObject

class CmdSetMoodInternal : ICmdCallback {
	override fun execute(player: Player, target: SWGObject?, args: String) {
		if (args.indexOf(' ') != -1) {
			return
		}

		player.creatureObject.moodId = args.toByte()
	}
}
